packages/expo-auth-session/src/providers/Google.ts

```diff
@@ -167,7 +168,7 @@ class GoogleAuthRequest extends AuthRequest {
    */
   async getAuthRequestConfigAsync(): Promise<AuthRequestConfig> {
     const { extraParams = {}, ...config } = await super.getAuthRequestConfigAsync();
-    if (config.responseType === ResponseType.IdToken && !extraParams.nonce && !this.nonce) {
+    if (config.responseType.split(' ').includes(ResponseType.Token) && !extraParams.nonce && !this.nonce) {
```

Comment:
```suggestion
    if (responseTypes.includes(ResponseType.Token) && !extraParams.nonce && !this.nonce) {
```

Let's reuse earlier set variable here.

Reply:
@Simek that was set in the constructor and isn't available here since it seemingly uses the `super.getAuthRequestConfigAsync` to retrieve the config